File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/sistemas/aprobacionniveles/dominio/dto/DtoNivelDetalle.ts
import { SyAprobacionNivelesDet } from '../syaprobacionnivelesdet';
import { DtoTablaEntero } from '../../../../shared/dominio/dto/DtoTablaEntero';

export class DtoNivelDetalle extends DtoTablaEntero {

    constructor() {
        super();
        this.listaDetalle = [];
    }

    listaDetalle: SyAprobacionNivelesDet[];
}

File: Administrativo/spring-erp-servicio/net/royal/spring/sistema/dominio/dto/DtoNivelDetalle.cs
using System;
using System.Collections.Generic;
using System.Text;
using net.royal.spring.framework.core.dominio.dto;

namespace net.royal.spring.sistema.dominio.dto
{
    public class DtoNivelDetalle: DtoTablaEntero
    {
        public List<SyAprobacionnivelesDet> ListaDetalle { get; set; }

        public DtoNivelDetalle()
        {
        }

        public DtoNivelDetalle(int codigo, String nombre)
        {
            this.codigo=codigo;
            this.nombre=nombre;
        }

    }
}
